### PR TITLE
Prac 08 feedback

### PR DESCRIPTION
**Pull request by ewabuczkowska**
Hi @PiperNeed4! Could you please review my structures and doc strings for this week's prac? Thank you!

### CODE REVIEW

File: prac_08/KivyDemos-master/convert_miles_km.kv
Comment (PiperNeed4) [inline prac_08/KivyDemos-master/convert_miles_km.kv:1]: The prac had required the program have a functioning button to convert miles to kilometers which is missing here

File: prac_08/KivyDemos-master/dynamic_labels.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+    def build(self):
+        """Build the Kivy GUI."""
+        self.title = "Dynamic Labels"
+        self.root = Builder.load_file('dynamic_labels.kv')
+        self.create_labels()
+        return self.root
+
+    def create_labels(self):
+        """Create buttons from data and add them to the GUI."""
```
Comment (PiperNeed4) [inline prac_08/KivyDemos-master/dynamic_labels.py:22]: This doc-string is inaccurate as the function creates labels

**Review by PiperNeed4 — COMMENTED** [review]
Other than some minor errors this is all very good work.

File: prac_08/KivyDemos-master/dynamic_labels.py (hunk 1 above)
Comment (ewabuczkowska) [reply prac_08/KivyDemos-master/dynamic_labels.py:22]: Thank you for noticing! I have changed it!

File: prac_08/KivyDemos-master/convert_miles_km.kv
Comment (ewabuczkowska) [reply prac_08/KivyDemos-master/convert_miles_km.kv:1]: Thank you Piper! I had missed that part!

**Pull request merged**